api/goldens/aptos_api__tests__accounts_test__test_get_account_resources_returns_empty_array_for_account_has_no_resources.json

```diff
@@ -1,13 +1,13 @@
 [
   {
-    "type": "0x1::coin::CoinInfo<0x1::aptos_coin::AptosCoin>",
```

Comment:
This shouldn't be converted back to test coin.  Could you rebase and regenerate these files?